test/test_um2netcdf.py

```diff
@@ -1177,3 +1187,79 @@ def test_convert_32_bit_with_float64(ua_plev_cube):
     ua_plev_cube.data = array
     um2nc.convert_32_bit(ua_plev_cube)
     assert ua_plev_cube.data.dtype == np.float32
+
+
+# fix forecast reference time tests
+@pytest.fixture
+def forecast_cube():
+    return DummyCube(item_code=999)
+
+
+@pytest.fixture
+def forecast_ref_time_coord():
+    # units data ripped from aiihca data file
+    unit = cf_units.Unit(unit="hours since 1970-01-01 00:00:00")
+    assert unit.calendar == um2nc.STANDARD
+
+    return iris.coords.DimCoord([-16383336.],
```

Comment:
Whats the significance of this number..?

Reply:
It looks like it's ~100 years before 1 Jan 1970:
```python
>>> -16382964.0 / 24 / 365  # convert to days & years
-1870.2013698630137
```
I've moved the value into a test fixture where it can be documented.